os/net/app-layer/snmp/snmp-ber.c

```diff
@@ -128,6 +128,11 @@ snmp_ber_encode_null(unsigned char *out, uint32_t *out_len, uint8_t type)
 unsigned char *
 snmp_ber_decode_type(unsigned char *buff, uint32_t *buff_len, uint8_t *type)
 {
+  if(*buff_len == 0)
+  {
```

Comment:
Style - opening brace in the same line with the if, like so:

`if(*buff_len == 0) {`